prow/config/config.go

```diff
@@ -1239,43 +1258,141 @@ type Branding struct {
 	HeaderColor string `json:"header_color,omitempty"`
 }
 
+// Note RerunAuthConfigs is a deprecated type.
```

Comment:
Please log a deprecation warning if this is set and add an announcement in prow/ANNOUNCEMENTS.md for this

Reply:
I was incorrect here, RerunAuthConfigs, isn't fully deprecated. It will still work, because we will convert that type into the slice. With this said, I opted not to include a warning for this, and reflect this in the announcements. In addition, references to deprecation have been removed.